### PR TITLE
Add eslint, move models to separate file, and remove module flag.

### PR DESCRIPTION
**Pull request by Osmose**
Moved the modules into their own file for readability and added ESLint so something yells at us when we mis-name variables.

I ran into so many problems with ESModule stuff while adding this that I gave up and replaced the imports/exports. ITS NOT READY YET HOMIE

### CODE REVIEW

**Comment by potch** [comment]
you're not ready yet

File: package.json [hunk 1]
```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "start": "onchange -ik '**/*.js' -- node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint .",
+    "dbsync": "node scripts/dbSync.js"
   },
-  "type": "module",
```
Comment (potch) [inline package.json:10]: booooooooo

File: package.json [hunk 2]
```diff
@@ -23,5 +24,8 @@
     "preact-render-to-string": "^5.1.19",
     "sequelize": "^6.6.2",
     "sqlite3": "^5.0.2"
+  },
+  "devDependencies": {
+    "eslint": "^7.25.0"
```
Comment (potch) [inline package.json:29]: thx you for this though

File: index.js [hunk 3]
```diff
@@ -164,7 +69,7 @@ const start = async () => {
       resave: false,
       saveUninitialized: true,
       cookie: {
-        maxAge: 60 * 525_602,
+        maxAge: 60 * 525602,
```
Comment (potch) [inline index.js:72]: heh

**Review by potch — APPROVED** [review]
danke

File: index.js (hunk 3 above)
Comment (Osmose) [reply index.js:72]: Eslint baybee

**Pull request merged**